README.md

```diff
@@ -8,12 +8,15 @@ OpenAG is an open-source client of the Half-Life promod Adrenaline Gamer, comple
 
 # Building
 ## Windows
-1. Get Visual Studio 2017 and CMake.
-2. `git submodule update --init`
-3. `mkdir build`
-4. `cd build`
-5. `cmake ..`
-6. `cmake --build . --config Release`
+1. Install Visual Studio 2017 from https://my.visualstudio.com/Downloads?q=Visual%20Studio%202017 (make sure you select Visual Studio 2017 Community version 15.9). In the Visual Studio Installer, select Desktop Development for C++ and not anything else.
+2. Install Git from https://git-scm.com/. Make sure you add it to PATH.
+3. Install CMake from https://cmake.org. Make sure you add it to PATH.
+4. Go to where you saved this cloned repo, right click ans select 'Git Bash Here.' In Git, type: `git submodule update --init`
```

Comment:
If you're explaining how to download and install VS, Git and CMake then you shouldn't skip the "cloning the repository" step.